Blog: Recommended Reading (#6525)
PostHog/posthog.com: contents/blog/recommended-reading.md

---
date: 2023-08-16
title: "PostHog's recommended reading for startup teams"
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author:
  - joe-martin
featuredImage: ../images/blog/super-hog-pink.png
featuredImageType: full
category: Inside PostHog
tags:
  - Guides
---

The PostHog team includes a number of voracious readers — we even have [our own book club](/handbook/people/training)! — so here’s a collection of the books our teams recommend. 

> 

Author: ```suggestion

There are thousands of books which will tell you how to build a product and launch a new startup, but it can be hard to know which are actually worth reading and which are full of generic advice (or written by AI). If you’re looking for inspiration, where should you turn?

The PostHog team includes a number of voracious readers — we even have [our own book club](/handbook/people/training)! — so here’s a collection of the books our teams recommend.

If you're looking for more startup advice or support, don't forget to check out [PostHog for Startups](/startups), our program for helping early-stage teams get to product market fit. Our startup program includes $50,000 in PostHog credit, exclusive events and promotional opportunities, and is available to any company less than 2 years old and with less than $5M in funding. 
```

![messy middle](../images/tracks/messy-middle.jpg)

## [The Messy Middle by Scott Belsky](https://www.goodreads.com/book/show/40179007-the-messy-middle)
**Recommended by [Charles Cook, Operations & Marketing](/community/profiles/254)**

If there’s one person whose book recommendations are really worth listening to, then it’s Charles. Not only is he the leader of both our Operations and Marketing teams — he’s _also_ the founder of our internal book club.

**Charles' review:** “I like The Messy Middle because, as the title suggests, it covers the really important interim period for startups that are starting to scale. Most startup type books cover either the very beginning when you’re validating your idea, or the later stages when you are 50+ people. This book is full of specific, actionable advice for the time in between those stages.”

> **Putting it into practice:** When it comes to ops, there’s no silver bullet. Instead, [you need a whole toolkit](/blog/startup-ops-toolkit) to help you solve messy problems!

> ![high output management](../images/tracks/high-output.jpg)

Author: ```suggestion
![high output management](../images/tracks/high-growth.jpg)
```

---
date: 2023-08-16

> title: "Recommended reading from the PostHog team"

Author: ```suggestion
title: "PostHog's recommended reading for startup teams"
```

![no rules rules](../images/tracks/no-rules-rules.jpg)
## [No Rules Rules by Reed Hastings](https://www.goodreads.com/book/show/49099937-no-rules-rules) 
**Recommended by [James Hawkins, CEO & Co-founder](/community/profiles/71)**

When it came to envisioning the sort of culture he wanted to build at PostHog, no was more influential for James as this exploration of Netflix’s company culture. PostHog doesn’t copy Netflix’s approach completely, but we definitely share some of the underlying philosophies. 

**James' review:** “No Rules Rules clarified how we should totally trust each person here. It also helped us realize just how much a smaller, stronger, and better paid group of people can accomplish... It’s just so counter to the way that other companies operate, where they focus on the whole function.”

> **Putting it into practice:** We validated a lot of Netflix’s rules when [hiring our first five employees](/blog/posthog-first-five) — and we’ve rolled them into [our overall culture](/about).

> ![getting real](../images/tracks/getting-real.png)

Author: ```suggestion
![getting real](../images/tracks/getting-real.jpg)
```

---
date: 2023-08-16
title: "Recommended reading from the PostHog team"
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author:
  - joe-martin
featuredImage: ../images/blog/super-hog-pink.png
featuredImageType: full
category: Inside PostHog
tags:
  - Guides
---

There are thousands of books which will tell you how to build a product and launch a new startup, but it can be hard to know which are actually worth reading and which are full of generic advice (or written by AI). If you’re looking for inspiration, where should you turn?

The PostHog team includes a number of voracious readers — we even have [our own book club](/handbook/people/training)! — so here’s a collection of the books our teams recommend. 

> 

Reviewer: I'd honestly consider cutting in the intro. Headline should be all the intro this article needs.

“I don’t agree about the importance of a dress code, though.”

> **Putting it into practice:** We encourage everyone at PostHog to [write a personal README](/team) about their style of work and how they like to collaborate and be communicated with. 

![From Switzerland](../images/tracks/from-switzerland.jpg)
## [From Switzerland by Counterprint](https://www.goodreads.com/book/show/60695040-from-switzerland)
**Recommended by [Lottie Coxon, Graphic Designer](/community/profiles/227)**

Lottie claims she doesn’t read very much, but when pressed for recommendations offered up From Switzerland as a representative of the entire Counterprint collection. Each book in the series collects work from some of the best agencies, designers and artists in each country — Switzerland, in this case.

> “This whole series is a good reset button for any designer stuck for inspiration,” says our artist-in-residence and the maestro behind [our mascot](/community/profiles/720). “It has a heavy focus on typographical print case studies, with experimental use of layout, negative space and color.”

Reviewer: ```suggestion
**Lottie's take:** “This whole series is a good reset button for any designer stuck for inspiration. It has a heavy focus on typographical print case studies, with experimental use of layout, negative space and color.
```

“I like The Messy Middle because, as the title suggests, it covers the really important interim period for startups that are starting to scale,” says Charles. “Most startup type books cover either the very beginning when you’re validating your idea, or the later stages when you are 50+ people. This book is full of specific, actionable advice for the time in between those stages.”

> **Putting it into practice:** When it comes to ops, there’s no silver bullet. Instead, [you need a whole toolkit](/blog/startup-ops-toolkit) to help you solve messy problems!

![high output management](../images/tracks/high-output.jpg)
## [High Output Management by Andrew Grove](https://www.goodreads.com/book/show/324750.High_Output_Management)
**Recommended by [Grace McKenzie, Finance & Operations Lead](/community/profiles/718)**

Andrew Grove was Intel’s third employee, third CEO, and the man who led the company to the world’s largest producer of semiconductors. He also somehow found time to write this book about organizational theory which is beloved by our chief organizer, Grace.

> “High Output Management is filled with light-touch and thoughtful management advice which can lead to increases in productivity,” says Grace. “In particular it advocates for the importance of gathering relevant information, making decisive decisions, and nudging teams towards a goal rather than setting a specific direction.”

Reviewer: ```suggestion
**Grace's take:** “High Output Management is filled with light-touch and thoughtful management advice which can lead to increases in productivity. In particular it advocates for the importance of gathering relevant information, making decisive decisions, and nudging teams towards a goal rather than setting a specific direction.”
```

> **Putting it into practice:** We’ve templated [our approach to goal-setting](/handbook/company/goal-setting) so that we can make fast decisions, and not waste time discussing formats. 

![no rules rules](../images/tracks/no-rules-rules.jpg)
## [No Rules Rules by Reed Hastings](https://www.goodreads.com/book/show/49099937-no-rules-rules) 
**Recommended by [James Hawkins, CEO & Co-founder](/community/profiles/71)**

> When it came to envisioning the sort of culture he wanted to build at PostHog, no other book was as influential for James as this exploration of Netflix’s company culture. PostHog doesn’t copy Netflix’s approach completely, but we definitely share some of the underlying philosophies. 

Reviewer: ```suggestion
When it came to envisioning the sort of culture he wanted to build at PostHog, no was more influential for James as this exploration of Netflix’s company culture. PostHog doesn’t copy Netflix’s approach completely, but we definitely share some of the underlying philosophies. 
```

> “No Rules Rules clarified how we should totally trust each person here,” says James. “It also helped us realize just how much a smaller, stronger, and better paid group of people can accomplish…It’s just so counter to the way that other companies operate, where they focus on the whole function.”

Reviewer: ```suggestion
**James' review:** “No Rules Rules clarified how we should totally trust each person here. It also helped us realize just how much a smaller, stronger, and better paid group of people can accomplish... It’s just so counter to the way that other companies operate, where they focus on the whole function.”
```

> **Putting it into practice:** We validated a lot of Netflix’s rules when [hiring our first five employees](/blog/posthog-first-five) — and we’ve rolled them into [our overall culture](/about).

![getting real](../images/tracks/getting-real.png)
## [Getting Real by Basecamp](https://basecamp.com/gettingreal)
**Recommended by [David Newell, Software Engineer](/community/profiles/860)**

Basecamp’s online manifesto about how they build products echoes a lot of the same principles we believe in, from the importance of iterative development to the focus on user feedback. David, who built several successful products before joining PostHog, thinks it’s a resource everyone can benefit from — not just engineers. 

> “It very simply talks about avoiding ‘work work’,” says David. “The focus on getting your product in front of customers as early as possible so that you avoid a bunch of implicit assumptions and sunk costs is particularly resonant.” 

Reviewer: ```suggestion
David's review: “It very simply talks about avoiding ‘work work’. The focus on getting your product in front of customers as early as possible so that you avoid a bunch of implicit assumptions and sunk costs is particularly resonant.” 
```

**Recommended by [Annika Schmid, Product Manager](/community/profiles/221)**

In addition to the The Mom Test, Annika recommends this book for its practical advice on gathering information from users and turning that information into actionable change. 

“It explains a method around continuously talking to users as opposed to doing interviews in big batches around specific topics, like larger organizations do,” says Annika. “I think it’s particularly useful for startups and it’s something we definitely do at PostHog.”

> **Putting it into practice:** Talking to users is great, but only if you invest the time in turning it into [actionable snapshots](/blog/interview-snapshot-guide).

![the mom test](../images/tracks/mom-test.jpg)
## [The Mom Test by Rob Fitzpatrick](https://www.goodreads.com/book/show/52283963-the-mom-test)
**Recommended by [basically everyone](/team)**

One of the most universally popular books among the team (shortly followed by The Design of Everyday Things by Dan Norman), The Mom Test has helped shape the way we think about product design and user feedback at a fundamental level. 

> “The Mom Test is an essential read on how to talk to potential customers and users,” says CEO & Co-founder James Hawkins in [his own guide on the topic](/blog/product-market-fit-game).
> 
> “The Mom Test is just a classic,” agrees Product Manager Annika Schmid. “Honestly, I recommend it for anyone who ever has to talk to users.” 

Reviewer: ```suggestion
James recommends it in his guide to [finding product-market fit]((/blog/product-market-fit-game), an Product Manager Annika Schmid is a big fan:

“The Mom Test is just a classic. Honestly, I recommend it for anyone who ever has to talk to users.” 
```

“Founding Sales is written primarily for Founders and CEOs, and covers all of the bases when it comes to sales,” says Simon Fisher. “I personally skipped some of the founder-centric stuff and focused on the advice for scaling Customer Success teams, which I found really useful.”

> **Putting it into practice:** We don’t have a Sales team, but [our Customer Success team](/handbook/small-teams/customer-success#mission) is very successful!

![saas playbook](../images/tracks/saas-playbook.jpg)
## [The SaaS Playbook by Rob Walling](https://www.goodreads.com/book/show/178816351-the-saas-playbook)
**Recommended by [Lior Neu-ner, Technical Content Marketer](/community/profiles/684)**

The SaaS Playbook isn’t your normal startup how-to guide, according to Lior. It doesn’t offer the usual hype around massive valuations and the importance of venture capital. Instead, it offers down-to-earth, practical advice for launching a new SaaS product in a repeatable way. 

> “It’s a very alternative guide to building a startup,” says Lior. “While most books center around hyper growth and fundraising, the SaaS Playbook teaches you how to achieve financial independence to build a sustainable and bootstrapped business.”

Reviewer: ```suggestion
**Lior's review:** “It’s a very alternative guide to building a startup. While most books center around hyper growth and fundraising, the SaaS Playbook teaches you how to achieve financial independence to build a sustainable and bootstrapped business.”
```

> > **Putting it into practice:** Product launches are a perpetual activity at PostHog, so our marketing team is always working in bootstrap mode — with help from [their favourite tool](/blog/posthog-marketing)

Reviewer: ```suggestion
> **Putting it into practice:** Product launches are a perpetual activity at PostHog, so our marketing team is always working in bootstrap mode — with help from [their favourite tool](/blog/posthog-marketing).
```

---

> date: 2023-08-16

Author: ```suggestion
date: 2023-08-22
```

**Recommended by [Annika Schmid, Product Manager](/community/profiles/221)**

In addition to the The Mom Test, Annika recommends this book for its practical advice on gathering information from users and turning that information into actionable change. 

“It explains a method around continuously talking to users as opposed to doing interviews in big batches around specific topics, like larger organizations do,” says Annika. “I think it’s particularly useful for startups and it’s something we definitely do at PostHog.”

> **Putting it into practice:** Talking to users is great, but only if you invest the time in turning it into [actionable snapshots](/blog/interview-snapshot-guide).

![the mom test](../images/tracks/mom-test.jpg)
## [The Mom Test by Rob Fitzpatrick](https://www.goodreads.com/book/show/52283963-the-mom-test)
**Recommended by [basically everyone](/team)**

One of the most universally popular books among the team (shortly followed by The Design of Everyday Things by Dan Norman), The Mom Test has helped shape the way we think about product design and user feedback at a fundamental level. 

> James recommends it in his guide to [finding product-market fit]((/blog/product-market-fit-game), and Product Manager Annika Schmid is a big fan:

Reviewer 2: ```suggestion
James recommends it in his guide to [finding product-market fit](/blog/product-market-fit-game), and Product Manager Annika Schmid is a big fan:
```
